python/lsst/meas/extensions/scarlet/scarletDeblendTask.py

```diff
@@ -525,6 +537,9 @@ def _addSchemaKeys(self, schema):
         self.parentNPeaksKey = schema.addField("deblend_parentNPeaks", type=np.int32,
                                                doc="Same as deblend_n_peaks, but the number of peaks "
                                                    "in the parent footprint")
+        self.parentNChildKey = schema.addField("deblend_parentNChild", type=np.int32,
+                                               doc="Same as deblend_nChild, but the number of "
+                                                   "deblended children from the parent footprint")
```

Comment:
I get the point of drawing an analogy to `deblend_nChild`, but perhaps it would be better to leave that as an afterthought as opposed to the lead-in (especially given how far up one has to look to find the doc for `deblend_nChild`)?  To fill in the blank, it does read rather strange to me:
"Same as "Number of children this object has (defaults to 0)", but the number of deblended children from the parent footprint"